jupyter_server/extension/application.py

```diff
@@ -377,9 +378,12 @@ def initialize(self, serverapp, argv=[]):
         """
         # Initialize ServerApp.
         self.serverapp = serverapp
-        
-        # Initialize the extension application
-        super(ExtensionApp, self).initialize(argv=argv)
```

Comment:
Will this be significant for mixin MROs? I.e. if a mixin is a sibling ancestor as `ExtensionApp`, will this make the order the siblings are declared (left-right / right-left) be dictated? Is it already dictated? If so, is it documented?

Reply:
Good question... I need to think about this some more.

Reply:
Did you figure something out? Punting is acceptable if deemed unimportant :)

Reply:
This PR significantly simplified how ExtensionApps are initalized and loaded. There is no need to call `super(...).initialize` anymore, since ExtensionApp's defer CLI parsing and config loading to the Server as it's parent app.